src/ObjectPool/src/IPooledObjectPolicy.cs

```diff
@@ -3,10 +3,23 @@
 
 namespace Microsoft.Extensions.ObjectPool
 {
+    /// <summary>
+    /// Represents a policy for managing pooled objects.
+    /// </summary>
+    /// <typeparam name="T">The type of object which is being pooled.</typeparam>
     public interface IPooledObjectPolicy<T>
     {
+        /// <summary>
+        /// Create a <typeparamref name="T"/>.
+        /// </summary>
+        /// <returns>The <typeparamref name="T"/> which was created.</returns>
         T Create();
 
+        /// <summary>
+        /// Return an object to the pool.
+        /// </summary>
+        /// <param name="obj">The object to return to the pool.</param>
+        /// <returns><code>true</code> if returned to the pool.</returns>
```

Comment:
This needs some more info.

You implement this method to run some processing when an object was returned to the pool. You can *reset* the state of an object. You also can return false from here if it's not possible/desirable for the pool to keep the object.